Library/Formula/anttweakbar.rb

```diff
@@ -1,5 +1,5 @@
 class Anttweakbar < Formula
-  desc "C/C++ library for adding GUIs to OpenGL apps"
+  desc "C/C++ library for adding GUIs 往 OpenGL apps"
```

Comment:
No, the audit bot found that the newline characters were introduced into the commit. I am so perplexed now.

Sorry for the PR.